lib/rubocop/cop/rspec/return_from_stub.rb

```diff
@@ -40,20 +40,21 @@ class ReturnFromStub < Cop
         MSG_BLOCK = 'Use block for static values.'
 
         def_node_search :contains_stub?, '(send nil? :receive (...))'
```

Comment:
There's one potential improvement I can think of - using `dynamic?` earlier so we don't have to perform searches in blocks that are not even related to stubs, like example groups. WDYT?